package/lib/clean.tests.js

```diff
@@ -583,4 +583,27 @@ describe('clean', function () {
       },
     });
   });
+
+  describe('with SimpleSchema.oneOf(String, Number, Date)', function () {
+    const oneOfSchema = new SimpleSchema({
+      field: {
+        type: SimpleSchema.oneOf(String, Number, Date),
+      },
+    });
+
+    function doTest(given, expected) {
+      const cleanObj = oneOfSchema.clean(given, { mutate: true });
+      expect(cleanObj).toEqual(expected);
+    }
+
+    it('should not convert a string', function () {
+      doTest({ field: 'I am a string' }, { field: 'I am a string' });
+    });
+    it('should not convert a number', function () {
+      doTest({ field: 12345 }, { field: 12345 });
+    });
+    it('should not convert a Date', function () {
+      doTest({ field: new Date(12345) }, { field: new Date(12345) });
+    });
```

Comment:
These tests are good, but I'd like to see the reverse tests, too. For example, if `Date` is removed from the `oneOf`, I guess a Date obj should be converted to String (the first type in the list).

Also it would be good to add at least one test that the cleaning works properly in a mongo modifier obj. Like your existing tests, but `{ $set: { field: 12345 } }` for example.